assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -356,7 +356,7 @@ che.infra.kubernetes.pvc.storage_class_name=
 che.infra.kubernetes.pvc.quantity=10Gi
 
 # Pod that is launched when performing persistent volume claim maintenance jobs on OpenShift
-che.infra.kubernetes.pvc.jobs.image=centos:centos7
+che.infra.kubernetes.pvc.jobs.image=registry.access.redhat.com/ubi8-minimal
```

Comment:
Maybe it's better to fix version instead of relying on the latest tag, which even is not set explicitly

Reply:
any reason to not use alpine there ?

Reply:
@sleshchenko  Sure will test ubi8 image with fix version and rebase the PR.
@benoitf  We thought ubi8-minimal as best suited replacement for centos7. Hence verified build with the same image.

Reply:
@sleshchenko Rebased PR. Please check.

Reply:
what about a more strict version `registry.access.redhat.com/ubi8-minimal:8.3-230`
CC @nickboldt

Reply:
@skabashnyuk Verified Che deployment with above mentioned image and it worked fine for s390x. I can rebase PR if change looks good.

Reply:
yes. please.
CC@nickboldt

Reply:
@skabashnyuk Rebased PR. Please check.

Reply:
+1 for UBI. Why make testing harder by having one image in upstream and a different one downstream?